tmt/steps/provision/testcloud.py

```diff
@@ -397,12 +397,15 @@ def start(self):
             self.info('progress', 'downloading...', 'cyan')
         try:
             self.image.prepare()
-        except FileNotFoundError:
-            raise ProvisionError(f"Image '{self.image.local_path}' not found.")
-        except testcloud.exceptions.TestcloudPermissionsError:
+        except FileNotFoundError as e:
+            raise ProvisionError(
+                f"Image '{self.image.local_path}' not found.\n {str(e)}"
+            )
+        except testcloud.exceptions.TestcloudPermissionsError as e:
             raise ProvisionError(
                 f"Failed to prepare the image. "
-                f"Check the '{TESTCLOUD_IMAGES}' directory permissions.")
+                f"Check the '{TESTCLOUD_IMAGES}' directory permissions.\n"
+                f"{str(e)}")
```

Comment:
Currently it prints 
```
Image '/var/tmp/tmt/testcloud/images/a.qcow2' not found.
 [Errno 2] No such file or directory: 'selinuxenabled'
```

Personally I'd prefix something so it prints e.g.
```
Image '/var/tmp/tmt/testcloud/images/a.qcow2' not found.
Raised exception was:  [Errno 2] No such file or directory: 'selinuxenabled'
```

@psss thoughts?

Reply:
Yes, something like that will be better. I think this is quite common scenario. Let's make handling of such exceptions a bit more comfortable. See 2d33f7e.